gcc/rust/expand/rust-expand-visitor.cc

```diff
@@ -230,37 +287,85 @@ ExpandVisitor::expand_inner_items (
 
 void
 ExpandVisitor::expand_inner_stmts (
-  std::vector<std::unique_ptr<AST::Stmt>> &stmts)
+  AST::BlockExpr &expr, std::vector<std::unique_ptr<AST::Stmt>> &stmts)
```

Comment:
```suggestion
  AST::BlockExpr &expr)
```
we can probably get away by just passing the `expr`, right? And the accessing `expr.get_stmts()`

Reply:
Fixed in e145c1eb94d76af6dee7f1ec0bc5f7c92f1d2779